README.md

```diff
@@ -291,11 +291,24 @@ Availability:
 
 ```python
 # input
-x = EnvironmentError('boom')        # also understands IOError and WindowsError
-raise EnvironmentError('boom')
+def f():
```

Comment:
I don't think you need a wrapping function here, right?

Reply:
nope. I can remove it